model/event/event_logger.go

```diff
@@ -19,9 +22,36 @@ func NewDBEventLogger(collection string) *DBEventLogger {
 	}
 }
 
-func (self *DBEventLogger) LogEvent(event EventLogEntry) error {
+func (self *DBEventLogger) LogEvent(event *EventLogEntry) error {
```

Comment:
does it make sense to change the reciver var name to something other than self, since we're here already?

Reply:
done